.travis.yml

```diff
@@ -6,3 +6,14 @@ env:
 install:
   - pip install -U tox
 script: tox
+deploy:
+  provider: pypi
+  distributions: "sdist bdist_wheel"
+  user: scrapinghub
+  password:
+    secure: bG1ycgBJrVfgBsHAwaWScL2V1x52/aZy2znE2kFzFfYq95HS51mu1GO2PKwpATJyMCs/xUIKGhiBtoo8LzNmNU2kPgarQFwzpLkx2ninOm+3uAwuIfGGWsonBW/h854n+cNwppJOlfEOxvMVerVVpdV4EFeJfEkVYascwOBbnqo=
+  on:
+    tags: true
+    all_branches: true
```

Comment:
it's best to remove this line, so that releases are made only for master branch.